docs/backends.rst

```diff
@@ -0,0 +1,25 @@
+Authentication Backends
+=====
+
+.. currentmodule:: authtools.backends
+
+django-authtools provides two authorization backend classes. These backends offer more customization
+for how your :class:`authtool.models.User` class is authenticated.
+
+.. class:: CaseInsensitiveEmailBackendMixin
+
+    This mixin simply calls the ``authenticate`` method of its superclass after lowercasing the
+    provided username. This superclass should be a user-defined or Django-provided authentication
+    backend, such as ``django.contrib.auth.backends.ModelBackend``.
+
+    .. warning:
+        Use of this mixin presupposes that all usernames are stored in their lowercase form, and
+        that there is no way to have usernames differing only in case. If usernames can differ in
+        case, this authentication backend mixin could cause errors in user authentication. It is
+        advised that you use this mixin in conjuction with the
+        ``CaseInsensitiveEmailUserCreationForm`` form provided in the ``forms`` module.
+
+.. class:: CaseInsensitiveEmailModelBackend
+    A subclass of the ``CaseInsentiveEmailBackendMixin`` with
+    ``django.contrib.auth.backends.ModelBackend`` as its chosen authentication backend superclass.
```

Comment:
This needs a sentence like: to use this, put this in your settings:

```python
AUTHENTICATION_BACKENDS = (
    'authtools.backends.CaseInsensitiveEmailModelBackend',
)
```

And then mention that if they have any other authentication backends that they need to mix the mixin into them.